src/ui/DialogSection.ts

```diff
@@ -18,13 +20,35 @@ export default abstract class Section {
    private generateDOM() {
       this.element = $('<section>');
 
-      if (this.title) {
-         let legendElement = $('<h2>');
-         legendElement.text(this.title);
-         legendElement.appendTo(this.element);
+      if (this.isCollapsible)
+      {
+         this.button = $('<button class="jsxc-collapsible-settings-button">'+this.title+'</button>');
+         this.button.on('click',(e)=>{
+            $(e.target).toggleClass('jsxs-collapsible-settings-button-active');
+            let content = $(e.target).next();
+            if (content.css('display') === 'block') {
```

Comment:
We can use css to hide/show the element. E.g. `.jsxs-collapsible-settings-button-active + div {display: block;}` or something should work.

Reply:
i refactored the jsxs-collapsible-settings-button class